www/js/share/services/websocket.service.js

```diff
@@ -535,6 +535,18 @@ angular
                                 localStorage.landingCompany = message.authorize.landing_company_name;
                                 appStateService.scopes = message.authorize.scopes;
                                 amplitude.setUserId(message.authorize.loginid);
+                                // update accounts from account list whenever authorize is received
+                                const accounts = localStorage.getItem('accounts') && JSON.parse(localStorage.getItem('accounts'));
```

Comment:
it's better to use `!_.isEmpty()` in first part of the AND statement. because if the content of the `account` has been set to `""` the second part of the code cause error.